trip_app/trips/tests/test_models.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from accounts.tests.user_factory import UserFactory
+
+from .trip_factory import TripFactory
+from ..models import Trip
+
+
+@pytest.fixture(
+    params=[0, 1, 10]
+)
+def passengers(request):
+    """ Create multiple users depending on params and return quantity"""
+    qty = request.param
+    return UserFactory.create_batch(size=qty, is_staff=False)
+
+
+@pytest.mark.django_db(transaction=True)
+class TestTrips:
+    def test_create_trip(self):
```

Comment:
Hm.. it is a reasonable question actually. Originally I was intended to test creation of a django model not a factory, but it seems that going with test library(factory boy) on such low level of testing makes us testing more a factory rather then model creation... But even if we abstract from the factory the question is should we ever need to test a django model? 
I realize that we should test custom logic that we add to a model such as creating/deleting of models, model methods, model manager methods, but not internal components of django. Doing so we can end up tests for every model field validation...
So make a long story short it seems that these tests are really excess and I should get reed of them.

Reply:
Yeah, I tend to think so too, i.e.:
1. anything straightforward (like factory boy) isn't worth testing
1. it might make sense to test something non-trivial within that straightforward, very selectively 
After all it'd be nice to have everything covered. But as we can see, even tests require additional resources/time for implementing/reviewing/further support.

Reply:
These tests should be removed now.